.ci/e2eTestingIngestManagerDaily.groovy

```diff
@@ -45,6 +45,7 @@ pipeline {
   parameters {
     choice(name: 'LOG_LEVEL', choices: ['INFO', 'DEBUG'], description: 'Log level to be used')
     choice(name: 'RETRY_TIMEOUT', choices: ['3', '5', '7', '11'], description: 'Max number of minutes for timeout backoff strategies')
+    booleanParam(name: "SKIP_GIT_CHECKS", defaultValue: "true", description: "If it's needed to check for Git changes to filter by modified sources")
```

Comment:
Default value is true. Maybe we can hide this param and force the variable to true. wdyt?